In [1]:
import joblib
import numpy as np
import pandas as pd

In [2]:
np_gib = joblib.load('gibberish_200text_780')
np_gib.shape

(200, 780)

### Create asap6 780-dim feature, including the gibberish (in total 1983 essays)

In [3]:
x = joblib.load('essay_ease10_sbert768_simbow_langerr_780_normalized_asap1')
y = joblib.load("score_asap1")

In [4]:
feats = np.concatenate((x, np_gib), axis=0)
feats.shape

(1983, 780)

In [5]:
joblib.dump(feats, 'essay_asap1_780_with200gibberish')

['essay_asap1_780_with200gibberish']

### Create asap6 score, with 0 (zero) scores for 200 gibberish (total 1983 esay scores)

In [6]:
zeros = np.zeros(200)
print(zeros.shape)
y = np.concatenate((y, zeros), axis=0)
print(y.shape)

(200,)
(1983,)


In [7]:
joblib.dump(y, 'score_asap1_with200gibberish')

['score_asap1_with200gibberish']